Libs/MRML/Core/vtkMRMLModelDisplayNode.h

```diff
@@ -73,23 +90,48 @@ class VTK_MRML_EXPORT vtkMRMLModelDisplayNode : public vtkMRMLDisplayNode
   /// \sa SetActiveScalarName()
   virtual void SetActiveAttributeLocation(int location);
 
-  /// Update the pipeline based on this node attributes
-  virtual void UpdatePolyDataPipeline();
+  /// Reimplemented to update scalar range accordingly
+  /// \sa SetActiveScalarName()
+  virtual void SetScalarRangeFlag(int flag);
+
+  /// Set whether to threshold the model display node.
+  /// \sa ThresholdEnabled, GetThresholdEnabled()
+  vtkSetMacro(ThresholdEnabled,bool);
+  vtkBooleanMacro(ThresholdEnabled,bool);
+
+  /// Get whether to threshold the model display node.
+  /// \sa ThresholdEnabled, SetThresholdEnabled(), ThresholdEnabledOn(),
+  /// ThresholdEnabledOff()
+  vtkGetMacro(ThresholdEnabled,bool);
+
+  /// Set the threshold range of the model display node.
+  /// \sa GetThresholdMin(), GetTresholdMax()
+  void SetThresholdRange(double min, double max);
+  void SetThresholdRange(double range[2]);
+
+  /// Get the threshold range of the model display node.
+  /// \sa SetThresholdRange()
+  void GetThresholdRange(double range[2]);
+  double GetThresholdMin();
+  double GetThresholdMax();
 
 protected:
   vtkMRMLModelDisplayNode();
   ~vtkMRMLModelDisplayNode();
   vtkMRMLModelDisplayNode(const vtkMRMLModelDisplayNode&);
   void operator=(const vtkMRMLModelDisplayNode&);
 
-  virtual void ProcessMRMLEvents(vtkObject *caller,
```

Comment:
@lassoan Once I put it back in `ProcessMRMLEvents, this warning is called 10 times when I load a volume: https://github.com/Slicer/Slicer/pull/647/files#diff-3e0d8da9d31fcf1472c2e8e9293b19a8R289

Should I just get rid of those warnings?